hashes/zkevm/src/keccak/mod.rs

```diff
@@ -8,806 +9,26 @@ use crate::{
         circuit::{Layouter, Region, Value},
         halo2curves::ff::PrimeField,
         plonk::{
-            Advice, Challenge, Column, ConstraintSystem, Error, Expression, Fixed, SecondPhase,
-            TableColumn, VirtualCells,
+            Challenge, Column, ConstraintSystem, Error, Expression, Fixed, TableColumn,
+            VirtualCells,
         },
         poly::Rotation,
     },
     util::expression::sum,
 };
-use halo2_base::halo2_proofs::{circuit::AssignedCell, plonk::Assigned};
 use itertools::Itertools;
 use log::{debug, info};
 use rayon::prelude::{IntoParallelRefIterator, ParallelIterator};
 use std::marker::PhantomData;
 
+pub mod cell_manager;
+pub mod keccak_packed_multi;
+pub mod param;
+pub mod table;
 #[cfg(test)]
 mod tests;
 pub mod util;
 
-use util::{
-    field_xor, get_absorb_positions, get_num_bits_per_lookup, into_bits, load_lookup_table,
```

Comment:
is it posible to do `git mv mod.rs keccak_packed_multi` so git records diff better?

Reply:
It cannot track the renaming: https://stackoverflow.com/questions/1094269/whats-the-purpose-of-git-mv
I tried to modify git commit message to:

```
# Changes to be committed:
#       new file:   src/keccak/cell_manager.rs
#       new file:   src/keccak/keccak_packed_multi.rs
#       renamed:    src/keccak_packed_multi/mod.rs -> src/keccak/mod.rs
#       new file:   src/keccak/param.rs
#       new file:   src/keccak/table.rs
#       renamed:    src/keccak_packed_multi/tests.rs -> src/keccak/tests.rs
#       renamed:    src/keccak_packed_multi/util.rs -> src/keccak/util.rs
#       modified:   src/lib.rs
```
But Git did't listen :(